plugins/catalog/src/index.ts

```diff
@@ -30,3 +30,5 @@ export {
   EntityHasSystemsCard,
   EntityLinksCard,
 } from './plugin';
+
+export { useOwnUser } from './components/useOwnUser';
```

Comment:
Don't think we want this tbh, should probably be moved to the `catalog-react` package? Tbh it's not a huge thing either so could perhaps copy it instead

Reply:
👍 will do!